arcade/drawing_support.py

```diff
@@ -108,7 +107,7 @@ def get_three_float_color(color: Color) -> Tuple[float, float, float]:
     :param Color color: Three or four byte tuple
     :return: Three floats as a RGB tuple
     """
-    if len(color) in (3, 4):
+    if len(color) in {3, 4}:
```

Comment:
This seems like a weird refactor.

Reply:
This is in fact slow:
https://stackoverflow.com/questions/45017774/python-set-vs-tuple-lookup-is-lookup-in-tuple-o1

Better use the tuple

Reply:
Yes, this is slower based on my tests as well. I'll change it back to a tuple. I don't know why it suggested this if a set is slower. 🤷

Reply:
> Yes, this is slower based on my tests as well. I'll change it back to a tuple. I don't know why it suggested this if a set is slower. 🤷

If set size is dynamic and big enough it will be faster. But not as a fixed size of 2 elements